packages/wdio-protocols/protocols/saucelabs.json

```diff
@@ -55,7 +29,7 @@
           " */"
         ],
         [
-          "// Get Performance Logs (coming soon)",
+          "// Get Performance Logs (needs capturePerformance capability)",
```

Comment:
Can we expand on this? Either by providing a link to our docs or showing an example of how to set it?

Reply:
added link to our performance docs